ui/packages/consul-ui/app/components/auth-form/index.hbs

```diff
@@ -75,8 +75,9 @@
       <em>Contact your administrator for login credentials.</em>
     </form>
 {{#if (env 'CONSUL_SSO_ENABLED')}}
+    {{!-- This `or` can be completely removed post 1.10 as 1.10 has optional params with default values --}}
```

Comment:
I'm pretty sure I want to backport this to 1.8.x, so I was planning on making this PR backport easily to 1.8.x and 1.9.x and then later on providing an additional PR just for 1.10.x (1.10.x has the `optionalParams` functionality which makes this additional `or` unnecessary)